docs/content/docs/usage/intl.md

```diff
@@ -40,183 +40,101 @@ Now create differents `.yml` files in a `translations` folder.
 If you want handle two languages, you have to create two files,
 with language key as filename.
 
-`en.yml`:
+`src/translation/en.yml`:
 
 ```yml
-locale: "en"
-
-messages:
-  phenomic-is-awesome: "Phenomic is awesome !"
+phenomic-is-awesome: "Phenomic is awesome !"
 ```
 
-`fr.yml`:
+`src/translation/fr.yml`:
 
 ```yml
-locale: "fr"
-
-messages:
-  phenomic-is-awesome: "Phenomic est génial !"
+phenomic-is-awesome: "Phenomic est génial !"
 ```
 
-From now you have 2 variations: with or without redux.
+## Initialize react-intl
 
-Without redux the thing will be pretty simple:
+We will need the `flat` lib to transform our translations files into flatten objects.
+
+```console
+npm i -S flat
+```
```

Comment:
Could you explain use case for flat? I add to open flat doc to understand the idea :)